.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-    - 0.10
```

Comment:
This is a number. That seems likely the reason that, we end up testing against Node.JS `0.1`.

We could use a string, but I've good experience with leaving the default.

If/when it fails, we can either fix it or pin the version.